.github/workflows/pull-request.yml

```
@@ -213,6 +213,27 @@ jobs:
         run: cp ./envFiles/.env.ci ./.env
       - name: Build talawa api compose testing environment
         run: docker compose build
+      - name: Wait for Postgres to be ready
+        run: |
+          echo "Waiting for Postgres..."
+          for i in {1..60}; do 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)